<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SecDialog {
border-image :url(:/renew.png);
}</string>
  </property>
  <widget class="QPushButton" name="submitpblm">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>140</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#submitpblm{
background-color: rgb(255, 255, 255);
	font: 75 16pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:4px;
}
#submitpblm:hover{
background-color:rgb(115, 255, 105);
font: 75 14pt &quot;Sitka Display&quot;;

}

</string>
   </property>
   <property name="text">
    <string>Submit a Problem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signout">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>190</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#signout{

	font: 75 16pt &quot;Sitka Display&quot;;
background-color: rgb(255, 255, 255);
border-width:0px;
border-radius:4px;
}
#signout:hover{
font: 75 14pt &quot;Sitka Display&quot;;
background-color:rgb(115, 255, 105);

}</string>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#back{
color:rgb(255, 255, 255);
font: 75 16pt &quot;Sitka Display&quot;;
background-color: rgb(0, 85, 127);
border-width:0px;
border-radius:4px;
}
#back::hover{
background-color:rgb(0, 96, 141);
font: 75 14pt &quot;Sitka Display&quot;;

}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>201</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{font: 18pt &quot;century gothic&quot;;
font-size:28px;
color:white;
background: #000000;
}
QLabel{
font: 18pt &quot;century gothic&quot;;
font-size:28px;
background:#ffffff;

}
QScrollArea QWidget
{  font: 18pt &quot;century gothic&quot;;
font-size:28px;

    background-color:rgb(0, 85, 127);
}


</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>199</width>
      <height>299</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>691</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{font: 18pt &quot;century gothic&quot;;
font-size:28px;
color:white;
background: #000000;
}
QLabel{
font: 18pt &quot;century gothic&quot;;
font-size:28px;
background:#ffffff;

}
QScrollArea QWidget
{  font: 18pt &quot;century gothic&quot;;
font-size:28px;

    background-color:rgb(0, 85, 127);
}

</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>689</width>
      <height>469</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>981</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>160</y>
       <width>471</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_8{
background-color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-1</y>
       <width>751</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="search">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#search{
border-width:0px;
border-radius:1px;
font: 20px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Search Location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchbttn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display</family>
     <pointsize>18</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#searchbttn{
color:rgb(255, 255, 255);
	font: 75 18pt &quot;Sitka Display&quot;;

background-color: rgb(0, 85, 127);
border-width:0px;
border-radius:1px;
}
#searchbttn::hover{
font: 75 14pt &quot;Sitka Display&quot;;

background-color:rgb(0, 111, 163);

}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1021</x>
     <y>71</y>
     <width>86</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;Login as :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1114</x>
     <y>71</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>941</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>660</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;© 2019 Team Noob.  All rights reserved.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
